sphinx/source/docs/user_guide/extensions.rst

```diff
@@ -15,10 +15,14 @@ that let you:
 * Create highly specialized models for domain-specific use cases
 
 You can make and use custom extensions with standard releases and don't need to
-set up a development environment or build anything from source. This is the
-easiest way to get involved in Bokeh development. You can try new features and
```

Comment:
FYI: I removed "This is the easiest way" because this would now contradict the new note ("is a highly advanced feature").

Reply:
I'm not sure they are contradictory. They are both an advanced feature for regular users, at the same time as being a good way for aspiring BokehJS devs to get started (since there is no need to set up a dev env)